ProcessMaker/processmaker
# BugFix: AddUserModal fields not clearing when closed.

In the Add User modal, the form fields do not clear when the modal is closed. This PR fixes that by refactoring the modal code to a vue bootstrap modal.

**To Test:**
Go to Admin
Click +User, notice the modal fields are empty
Close the modal by pressing the X
Click +User again, notice the modal fields are empty
Fill out all the fields, close the modal
Click +User one last time

Expected Behavior
The fields should be cleared regardless of what method the modal is closed

closes #2527
